From 8552d42e7a3884a4d1e052d9d772bbf3b4e4f02e Mon Sep 17 00:00:00 2001
From: pexcn <i@pexcn.me>
Date: Tue, 5 May 2020 22:46:20 +0800
Subject: [PATCH] build with gcc O3 optimizations

---
 Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index 06cb6de..eeeacf6 100644
--- a/Makefile
+++ b/Makefile
@@ -1,12 +1,12 @@
 CC = gcc
-CFLAGS = -std=c99 -Wall -Wextra -O2 -pthread
+CFLAGS = -std=c99 -Wall -Wextra -O3 -pthread
 LIBS = -lm
 SRCS = ipt2socks.c logutils.c lrucache.c netutils.c protocol.c
 OBJS = $(SRCS:.c=.o)
 MAIN = ipt2socks
 DESTDIR = /usr/local/bin
 
-EVCFLAGS = -w -O2
+EVCFLAGS = -w -O3
 EVSRCFILE = libev/ev.c
 EVOBJFILE = ev.o
 
-- 
2.26.0

